frontend/src/metabase/core/components/AutocompleteInput/AutocompleteInput.stories.tsx

```diff
@@ -14,7 +14,8 @@ const Template: ComponentStory<typeof AutocompleteInput> = args => {
       {...args}
       value={value}
       onChange={setValue}
-      placeholder={"Fruits"}
+      onOptionClick={undefined}
```

Comment:
This seems not be needed. If it is for some reason, it should be fixed imo.

Reply:
This is a really weird one, but Storybook will always include function stubs in `args`, and since the component relies on `onOptionClick` being undefined for default behavior, it was causing issues. Not sure what a better solution would be here.

Reply:
Got it